src/bank_machine/content.clj

```diff
@@ -80,14 +60,11 @@
           account-digit-parts-collection []
         ]
     (if (< i 3) ;; skip the 4th line
-      (do
-        (def account-line-digit-parts-collection (group-account-line-into-digit-parts (get account i)))
-        (recur (inc i)(conj account-digit-parts-collection account-line-digit-parts-collection))
-      )
-      account-digit-parts-collection
-    )
-  )
-)
+      (let [account-line-digit-parts-collection (group-account-line-into-digit-parts (get account i))]
```

Comment:
Typical to use `let` in these cases. The `def` technically defined a globally scoped variable that could be accessed outside this code (which you aren't doing, anyway).